addons/survey/models/survey_question.py

```diff
@@ -265,6 +265,17 @@ def _compute_is_scored_question(self):
             else:
                 question.is_scored_question = False
 
+    # ------------------------------------------------------------
+    # CRUD
+    # ------------------------------------------------------------
+
+    @api.ondelete(at_uninstall=False)
+    def _unlink_prevention_of_survey_question(self):
+        """prevent deletion of question when the session is in progress"""
```

Comment:
I think we can also drop the comment in favor of a more explicit method name:

```suggestion
    def _unlink_except_live_sessions_in_progress(self):
```